extension/opampextension/README.md

```diff
@@ -25,6 +25,7 @@ The following settings are optional:
   - `ws`: The OpAMP websocket transport settings.
     - `tls`: TLS settings.
     - `headers`: HTTP headers to set.
+    - `polling_interval`: The interval at which the extension will poll the server. Only applicable to HTTP client. Defaults to 30s.
```

Comment:
The way the README is formatted here seems to imply that polling_interval is a `ws` setting, but it's http only. Can we nest it under a new `http` section?

Reply:
We could, not sure though how we then want to show both since now we show both in single section and WS vs. HTTP is matter of configuring the transport setting. I'll try to adjust it.